pallets/reactions/src/benchmarking.rs

```diff
@@ -0,0 +1,104 @@
+//! Reactions pallet benchmarking.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+use frame_benchmarking::{benchmarks, whitelisted_caller};
+use frame_support::{dispatch::DispatchError, traits::Currency};
+use frame_system::RawOrigin;
+use pallet_posts::{Post, PostExtension};
+use pallet_spaces::types::Space;
+use sp_std::vec;
+use subsocial_support::Content;
+
+fn create_dummy_space<T: Config>(
+    origin: RawOrigin<T::AccountId>,
+) -> Result<Space<T>, DispatchError> {
+    let space_id = pallet_spaces::NextSpaceId::<T>::get();
+
+    pallet_spaces::Pallet::<T>::create_space(origin.clone().into(), Content::None, None)
+        .map_err(|e| e.error)?;
+
+    let space = pallet_spaces::SpaceById::<T>::get(space_id)
+        .ok_or(DispatchError::Other("Space not found"))?;
+
+    Ok(space)
+}
+
+fn create_dummy_post<T: Config>(origin: RawOrigin<T::AccountId>) -> Result<Post<T>, DispatchError> {
+    let post_id = pallet_posts::NextPostId::<T>::get();
+    let space = create_dummy_space::<T>(origin.clone())?;
+
+    pallet_posts::Pallet::<T>::create_post(
+        origin.clone().into(),
+        Some(space.id),
+        PostExtension::RegularPost,
+        Content::None,
+    )?;
+
+    let post =
+        pallet_posts::PostById::<T>::get(post_id).ok_or(DispatchError::Other("Post not found"))?;
+
+    Ok(post)
+}
+
+fn create_dummy_post_reaction<T: Config>(
+    origin: RawOrigin<T::AccountId>,
+) -> Result<(Post<T>, Reaction<T>), DispatchError> {
+    let post = create_dummy_post::<T>(origin.clone())?;
+    let reaction_id = NextReactionId::<T>::get();
+
+    Pallet::<T>::create_post_reaction(origin.clone().into(), post.id, ReactionKind::Upvote)?;
+
+    let reaction =
+        ReactionById::<T>::get(reaction_id).ok_or(DispatchError::Other("Reaction not found"))?;
+
+    Ok((post, reaction))
+}
+
+benchmarks! {
+    create_post_reaction {
+        let origin = RawOrigin::Signed(whitelisted_caller());
+        let post = create_dummy_post::<T>(origin.clone())?;
+        let reaction_kind = ReactionKind::Upvote;
+        let reaction_id = NextReactionId::<T>::get();
+
+    }: _(origin, post.id, reaction_kind)
+    verify {
+        ensure!(ReactionIdsByPostId::<T>::get(post.id) == vec![reaction_id], "Reaction is not found");
+        ensure!(
+            ReactionById::<T>::get(reaction_id)
+                .expect("Reaction not found")
```

Comment:
I believe that we should improve the description for test.

Maybe 
First: "Reaction is not found by postId."
Seconde: "Reaction is not found."